### DISCUSSION TITLE
[DEV] How do I modify boss spells not in database

### DISCUSSION THREAD

**Discussion by cts17** [azerothcore/azerothcore-wotlk#3904] [General]
If a particular spell from a boss fight is doing too much (or too little) damage, how does one go about modifying that? I've been told to look in spell.dbc and so forth, but not all spells seem to show up in that database.

**Comment by Kitzunu** [comment]
Check creature_template there are some damage modifiers there

**Comment by cts17** [comment]
Don't those seem to modify the damage done by the boss rather than the scripted spells?

**Comment by Si1ker** [comment]
@Kitzunu DamageModifier in creature_template only modifies the damage based on mindamage and maxdamage. In resume, it is a multiplier that will affect the melee creature attacks, nothign to do with spells.

@cts17 If you want to modify spells, that would be in the spell dbc , be careful it is not recommended at all trying to modify it.
